GDJS/Runtime/pixi-renderers/runtimescene-pixi-renderer.js

```diff
@@ -133,11 +133,14 @@ gdjs.RuntimeScenePixiRenderer.prototype.setLayerIndex = function (
   /** @type {gdjs.LayerPixiRenderer} */
   // @ts-ignore - assume the renderer is the correct one
   var layerPixiRenderer = layer.getRenderer();
+
+  /** @type {PIXI.Container | ?PIXI.Sprite} */
   var layerPixiObject = layerPixiRenderer.getRendererObject();
 
   if (layer.isLightingLayer())
     layerPixiObject = layerPixiRenderer.getLightingSprite();
 
+  if (!layerPixiObject) return;
   if (this._pixiContainer.children.indexOf(layerPixiObject) === index) return;
 
   this._pixiContainer.removeChild(layerPixiObject);
```

Comment:
@4ian, in the case of the lighting layer, the layer's pixi container is already removed (and replaced with lighting sprite). This way, we could handle all the cases for hot-reloading. We can reorder lighting layer as well, which will basically change the index of the sprite, instead of the container (which wouldn't even exist in the scene's container in the first place).

Reply:
That sounds fine, it's also the reason for calling the method "getRendererObject" initially - the layer can be a PIXI.Container, but also a PIXI.Sprite.